app/controllers/concerns/ndjson_upload_concern.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Shared controller methods for uploading ndjson files for harvesting
+module NdjsonUploadConcern
```

Comment:
Can we make this a class instead?  Then we can pass in the required URL... That makes it easier to see what the dependencies of this code is.

Reply:
sure, I like that pattern. will do

Reply:
should it still live in `concerns/`?

Reply:
No, probably app/services or similar.